packages/splitText/src/splitIntoWords.mjs

```diff
@@ -1,2 +1,27 @@
-// Hypothesis: A word consists of non-space characters, followed by any number of space characters.
-export default (text) => text.match(/\s*\S+\s*/g) || [text];
+// Break words at spaces and dashes.
+// Why a dashes? If we don't, long words connected by dashes (especially in German) will blow
+// the layout as lines become longer than the layout might be wide.
+export default (text) => {
+    // This splits text into words and whatever's inbetween; 'hello-world there' becomes
+    // ['hello', '-', 'world', ' ', 'there']
+    // Parens are necessary to include the boundaries in the result
```

Comment:
Diesen Satz verstehe ich nicht (deepl auch nicht...)

Reply:
[Parens = Klammer](https://dict.leo.org/englisch-deutsch/parens). Es braucht also die Klammern in der Regex, damit die Orte, an denen wir splitten (d.h. die Word Boundaries) eingeschlossen werden im Resultat. Ich formulier's noch etwas um.